qtawesome/iconic_font.py

```diff
@@ -632,8 +632,15 @@ def _install_fonts(self, fonts_directory):
                         GFRI_ISTRUETYPE)
                     if is_truetype:
                         fontname += ' (TrueType)'
-                    with winreg.OpenKey(winreg.HKEY_CURRENT_USER, FONTS_REG_PATH, 0,
-                                        winreg.KEY_SET_VALUE) as key:
-                        winreg.SetValueEx(key, fontname, 0, winreg.REG_SZ, filename)
+                    try:
+                        with winreg.OpenKey(winreg.HKEY_CURRENT_USER, FONTS_REG_PATH, 0,
+                                            winreg.KEY_SET_VALUE) as key:
+                            winreg.SetValueEx(key, fontname, 0, winreg.REG_SZ, filename)
+                    except FileNotFoundError:
```

Comment:
Perhaps we should change this to `except Exception` just in case other errors could be raised here?

Of course, if the only errors what `OpenKey` and `SetValueEx` are `FileNotFoundError`, then there's no need of that.

Reply:
Checking I think [they raise](https://docs.python.org/3/library/winreg.html#winreg.OpenKey) an `OSError` subclass if something happens so this will end up being:
```suggestion
                    except OSError:
```
Should I change it from `FileNotFoundError` to `OSError`?

Reply:
Ok, sounds good to me.